packages/components/src/Components/Maintenance/Maintenance.js

```diff
@@ -9,7 +11,13 @@ import { HourglassHalfIcon } from '@patternfly/react-icons';
 
 import './maintenance.scss';
 
-const Maintenance = ({ startTime, endTime, timeZone, description }) => {
+const Maintenance = ({ utcStartTime, utcEndTime, startTime, endTime, timeZone, description, redirectLink}) => {
```

Comment:
make sure to take in a className and a props spread here in case people need anything else

Reply:
Should be g2g now!